code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -83,14 +85,37 @@
 /mob/living/simple_animal/construct/cultify()
 	return
 
-/mob/living/simple_animal/construct/New()
+/mob/living/simple_animal/construct/New(var/mob/living/carbon/user)
 	..()
+
+	if(istype(user, /mob/living/carbon))
```

Comment:
This shouldn't be here, put this logic into a different proc and call it after creating the construct.